controllers/tab/issueEntry/form/IssueEntryPublicationMetadataForm.inc.php

```diff
@@ -284,11 +289,13 @@ function execute($request) {
 
 			if ($issue) {
 
+				// set publication date
+				$submission->setDatePublished($this->getData('datePublished') ? $this->getData('datePublished') : Core::getCurrentDate());
```

Comment:
If you remove a submission (or a version of a submission) from an issue, the (whole) article does not show at the frontend. I think this makes sense. Also if you add the article again all old dates are still there, which is also good, I think.